requirements.txt

```diff
@@ -0,0 +1 @@
+redis
```

Comment:
Maybe not a bad idea to pin it to a specific version - e.g. `redis>=2.10.6,<2.11`.

This way the pack installation will be reproducible and there will be no surprises if library is updated or similar.